src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+
+type Props = {
+  addMovie: (arg0: Movie) => void | null
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [loading, setLoading] = useState<boolean>(false);
```

Comment:
it is not necessary for primitive types
```suggestion
  const [loading, setLoading] = useState(false);
```